app/utils/hentFraApi.ts

```diff
@@ -1,6 +1,14 @@
 import søkerMock from '~/mock/søkerMock';
+import { fetchWithToken } from '~/server/authorization';
+import { EMiljø } from '~/typer/miljø';
+import { ISøker } from '~/typer/søker';
 
-export const hentSøker = () => {
-  const søker = søkerMock;
-  return søker;
+const API_URL_BACKEND: string = 'http://localhost:8099/api/oppslag/soker';
```

Comment:
Vi skal ikke hente fra localhost i produksjon, men fra `https://nav-familie-endringsmelding-api.fly.dev/`

Reply:
Kommer den fra `bekk/nav-familie-endringsmelding-api` repoet? Er den deployed? 🥸

Reply:
Løsning for dette i commit 566eda8 :
```javascript
export const hentSøker = async (request: Request): Promise<ISøker> => {
  switch (process.env.ENV) {
    case EMiljø.LOKAL:
      return (await fetchWithToken(request, LOCAL_URL_BACKEND)).json();
    case EMiljø.PRODUKSJON:
      return (await fetchWithToken(request, API_URL_BACKEND)).json();
    default:
      return Promise.resolve(søkerMock);
  }
};

```

Reply:
Det er oppe og kjører 😊 
https://nav-familie-endringsmelding-api.fly.dev/api/ping